.env.example

```
@@ -0,0 +1,3 @@
+EXPO_PUBLIC_API_KEY=ta_cle_api_ici
+EXPO_PUBLIC_API_URL=https://ton-serveur.com/api
+SENTRY_AUTH_TOKEN=ton_token_sentry_ici
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)